R/RMEvents_sko.R

```diff
@@ -15,6 +15,14 @@
 #' @examples
 RMevents_sko <- function(df,ieHr=6,rainthresh=5.1,rain="rain",time="pdate"){
   
```

Comment:
@rbcarvin : I added 2 checks here. 1 is to make sure the `time` column is correctly defined. The other is to check if all the dates are NA. How about you merge this pr and try to do a little debugging on the cooperator data to try to figure out why it's not working.